ui/tests/integration/components/sync/secrets/page/overview-test.js

```diff
@@ -53,135 +53,263 @@ module('Integration | Component | sync | Page::Overview', function (hooks) {
       );
   });
 
-  test('it should render landing cta component for enterprise with the secrets sync feature', async function (assert) {
-    this.destinations = [];
+  test('it should render header, tabs and toolbar for overview state', async function (assert) {
     await this.renderComponent();
 
     assert.dom(title).hasText('Secrets Sync', 'Page title renders');
-    assert.dom(cta.button).hasText('Create first destination', 'CTA action renders');
-    assert.dom(cta.summary).exists('CTA renders');
+    assert.dom(breadcrumb).exists({ count: 1 }, 'Correct number of breadcrumbs render');
+    assert.dom(breadcrumb).includesText('Secrets Sync', 'Top level breadcrumb renders');
+    assert.dom(cta.button).doesNotExist('CTA does not render');
+    assert.dom(tab('Overview')).hasText('Overview', 'Overview tab renders');
+    assert.dom(tab('Destinations')).hasText('Destinations', 'Destinations tab renders');
+    assert.dom(overview.createDestination).hasText('Create new destination', 'Toolbar action renders');
   });
 
-  test('it should render landing cta component for community', async function (assert) {
-    this.version.type = 'community';
-    this.version.features = [];
-    this.destinations = [];
+  module('community', function (hooks) {
```

Comment:
i apologize for how hard these changes are to read through. much of it is me reorganizing existing tests so it's clearer which scenarios we're covering. here's a helpful screenshot of how the test modules are organized. open to feedback!

<img width="793" alt="Screenshot 2024-03-27 at 7 53 19 PM" src="https://github.com/hashicorp/vault/assets/903288/88d072db-e07e-4936-b76c-ded909a0ed60">

Reply:
I agree with all of this, *except* I like keeping the assertion messages because in the larger tests it is easier to figure out what was happening in the lead-up to the part where it failed. Otherwise the test suite just says "ok" unless it fails, which is not as helpful. But, this is sort of mitigated by the smaller test files, so I think both ways work

Reply:
> Otherwise the test suite just says "ok" unless it fails
oh, that's a great point. i'm used to `cmd + F`ing in test files to figure out the failure and what was happening leading up to it. let's talk about this during our team pattern chat today!